help/blocks/search/search.css

```diff
@@ -1,29 +1,130 @@
 .search {
-  margin-top: 24px;
+  margin-top: 1.5rem;
+  position: relative;
 }
 
 .search-outer {
-  max-width: 1280px;
+  max-width: 75rem;
   box-sizing: border-box;
   padding: 2px;
   position: relative;
 }
 
+.search .error-messages {
+  margin: 0.25rem 0.5rem;
+  padding-bottom: 2rem;
+}
+
+.search .results-list-title {
+  font-family: var(--body-font-family-bold-normal);
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  display: inline;
+}
+
+.search .results-list-link:visited .results-list-title {
+  color: #064384;
+}
+
+.search .results-list {
+  margin: 0 1rem 1rem;
+}
+
+.search .results-list-item {
+  border-bottom: 1px solid var(--body-background-color);
+  padding: 2rem 0;
+}
+
+.search .results-list-link {
+  margin-bottom: 0.5rem;
+}
+
+.search .results-list-link:focus {
+  color: var(--button-hover-color);
+  text-decoration: underline;
+}
+
+.search .results-list-url,.results-list-description {
+  font-size: .875rem;
+  line-height: 1.25rem;
+  word-wrap: break-word;
+}
+
+.search .results-list-url {
+  margin-top: 0.5rem;
+}
+
 .search-input {
   font-family: inherit;
   font-size: 1.125rem;
   line-height: 1.5rem;
   width: 100%;
-  max-height: 75px;
+  max-height: 4.688rem;
   border: 0;
   color: var(--search-text-color);
-  padding: 16px 48px 24px 16px;
+  padding: 1rem 3rem 1.5rem 1rem;
   box-sizing: border-box;
   max-width: unset;
   border-radius: unset;
   margin: 0;
 }
 
+.search ul.pagination {
+  display: flex;
+  justify-content: center;
+  margin-top: 3.125rem;
+  margin-bottom: 6.25rem;
+  list-style: none;
+  flex-wrap: wrap;
+}
+
+.search .pagination a {
+  font-size: 16px;
+  color:var(--link-color);
+  padding: 0.25rem 1.25rem;
+  text-decoration: none;
+  font-weight: 700;
+}
+
+.search .pagination .notvisible {
+  display: none;
+}
+
+.search .pagination li:nth-of-type(2) a {
+  border-left: 1px solid var(--overlay-background-color);
+}
+
+.search .pagination li.page {
+  background: url('/help/icons/search-arrow.svg') no-repeat center;
+}
+
+.search .pagination li.prev.page {
```

Comment:
pagination should be center aligned, currently it seems going towards right more.